app/src/main/java/org/simple/clinic/summary/PatientSummaryConfig.kt

```diff
@@ -17,12 +19,14 @@ data class PatientSummaryConfig(
       val bloodSugarEditableDurationConfig = configReader.long("blood_sugar_editable_duration_in_seconds", 3600)
       val isDiabetesEnabled = configReader.boolean("diabetes_enabled", false)
       val numberOfMeasurementsToDisplay = configReader.long("number_of_measurements_to_display_in_summary", 3)
+      val isBloodSugarEditable = BuildConfig.DEBUG
```

Comment:
We try to avoid using `BuildConfig.DEBUG` directly in our modules. The way we generally do separate modules for `DEBUG` and `RELEASE` builds is by cloning the module in the `debug` and `release` source sets. Could you change to this method of doing this, please?

We can pair on this if you're not entirely sure of how we generally do this.